package.json

```diff
@@ -6,7 +6,17 @@
   "scripts": {
     "prepare": "husky install"
   },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node-esm",
+      "js": "ts-node-esm"
+    }
+  },
   "devDependencies": {
+    "@types/express": "^4.17.20",
```

Comment:
Remove @types/express and @types/node. Those only go in each exercise, not the root.